bindings/python/capstone/arc.py

```diff
@@ -0,0 +1,40 @@
+# Capstone Python bindings, by R33v0LT <sibirtsevdl@gmail.com>
+
+import ctypes
+from . import copy_ctypes_list
+from .arc_const import *
+
+
+class ARCOpValue(ctypes.Union):
+    _fields_ = (
+        ('reg', ctypes.c_uint),
+        ('imm', ctypes.c_int64)
+    )
+
+
+class ARCOp(ctypes.Structure):
+    _fields_ = (
+        ('type', ctypes.c_uint),
+        ('value', ARCOpValue),
+        ('access', ctypes.c_uint8)
```

Comment:
```suggestion
        ('access', ctypes.c_int)
```

It was an enum in `arc.h`. Please either make it an `uint8` or change it here to an `int`